react-backend/client/src/components/authentication/pswdresetcb.js

```diff
@@ -14,17 +15,17 @@ class CBPswdReset extends Component {
     };
   }
 
-  handleChange = e => {
-    let newState = {};
-    newState[e.target.name] = e.target.value;
-    this.setState(newState);
-  };
-
   setError(messagesArray) {
     this.setState({ error: messagesArray });
   }
 
-  handleSubmit = e => {
+  handleChange = (e) => {
+    const newState = {};
+    newState[e.target.name] = e.target.value;
+    this.setState(newState);
+  };
+
+  handleSubmit = (e) => {
```

Comment:
Is there a convention your following for reordering methods?

Reply:
I do what the linter tells me to. I think the logic it follows is lifecycle methods first, auto-bound methods last, but if you're really interested, you can lookup the react/sort-comp linting rule.